text/000x-assethub.md

```diff
@@ -0,0 +1,92 @@
+| attributeDepositBase      | 0.00666666666 KSM                | 0.15333333318 $        | 3.333333333 $                 | 0.000666666666 KSM        | 0.015333333318 $                 | 0.3333333333 $                         |
+
+```
+> Note: This is only a proposal for change and it can be modified upon additional conversation.
+
+## Drawbacks
+The contemplation of modifying deposit requirements necessitates a balanced assessment of the potential drawbacks. Highlighted below are cogent points extracted from the discourse on the [Polkadot Forum conversation](https://forum.polkadot.network/t/polkadot-assethub-high-nft-collection-deposit/4262), which provide critical perspectives on the implications of such changes:
+
+> But NFT deposits were chosen somewhat arbitrarily at genesis and it’s a good exercise to re-evaluate them and adapt if they are causing pain and if lowering them has little or no negative side effect (or if the trade-off is worth it).
+>  -> joepetrowski
+
+> Underestimates mean that state grows faster, although not unbounded - effectively an economic subsidy on activity. Overestimates mean that the state grows slower - effectively an economic depressant on activity.
+>  -> rphmeier
+
+> Technical: We want to prevent state bloat, therefore using state should have a cost associated with it.
+>  -> joepetrowski
+
+
+## Testing, Security, and Privacy
+
+## Performance, Ergonomics, and Compatibility
+
+### Performance
+This change is not expected to have a significant impact on the overall performance of the Polkadot Asset Hub. However, it is crucial to monitor the network closely, especially in the initial stages after implementation, to identify and mitigate any potential issues.
+ 
+Additionally, a supplementary proposal aims to augment the network's adaptability:
+
+> Just from a technical perspective; I think the best we can do is to use a weak governance origin that is controlled by some consortium (ie. System Collective).
+> This origin could then update the NFT deposits any time the market conditions warrant it - obviously while honoring the storage deposit requirements.
+> To implement this, we need RFC#12 and the Parameters pallet from @xlc.
+>  -> OliverTY
+
+This dynamic governance approach would facilitate a responsive and agile economic model for deposit management, ensuring that the network remains both accessible and robust in the face of market volatility.
+
+### Ergonomics
+The proposed change aims to enhance the user experience for artists, making Polkadot more accessible and user-friendly.
+
+### Compatibility
+The change doesn't impact compatibility.
+
+## Unresolved Questions
+- Determining the optimal new deposit amount that reduces the barrier to entry while preventing state bloat remains a subject for debate. What is the balance that allows for inclusivity without compromising the network's efficiency and security?
+
+## Future Directions and Related Material
```

Comment:
I think a good future direction would be to establish some kind of process on how to agree on these values.  
Otherwise we will have the same discussion every 3-6 months i guess.

Reply:
The proposed prices have been adjusted so that they are not too low, but still have a reasonable maximum at a historical maximum of prices of DOT and KSM.